node-it/src/test/scala/com/wavesplatform/it/api/SyncHttpApi.scala

```diff
@@ -743,13 +744,32 @@ object SyncHttpApi extends Assertions {
       maybeWaitForTransaction(sync(async(n).grpc.broadcast(tx,proofs)), waitForTx)
     }
 
+
     def broadcastSponsorFee(sender: KeyPair,
                             minFee: Option[Amount],
                             fee: Long,
                             version: Int = 1,
                             waitForTx: Boolean = false): PBSignedTransaction = {
       maybeWaitForTransaction(sync(async(n).grpc.broadcastSponsorFee(sender, minFee, fee, version)), waitForTx)
     }
+
+    def broadcastInvokeScript(caller: KeyPair,
```

Comment:
Move the whole NodeExtGrpc to another file, please

Reply:
done